questions/views.py

```diff
@@ -59,7 +59,7 @@ def show_questions(request, form=None, current_category=0):
         .annotate(answered=Exists(their_answers_for_question)) \
         .annotate(voted=Exists(their_votes_for_question)) \
         .order_by('-num_votes','category__number')
-    if current_category:
+    if current_category == 0:
```

Comment:
? it was right before; this will return nothing since I doubt anything has an id 0

Reply:
amended to `if category != 0:` I can't pass none from the select, so I pass 0, which is also the default. `if != 0` then the cat filter is applied.